### ISSUE TITLE
Custom voice data? 

### ISSUE THREAD

**Issue by MartinSokolov15** [chamara96/voice-command-rnn#1]
So, can i train my own voice data ?

**Comment by chamara96** [comment]
@MartinSokolov15 Yes, you can. We've created our own single-word dataset and then used transfer learning to adapt the original model.

**Comment by MartinSokolov15** [comment]
> @MartinSokolov15 Yes, you can. We've created our own single-word dataset and then used transfer learning to adapt the original model.

Can i use this model to control Servos with my own voice commands ?
